spectator/__init__.py

```diff
@@ -7,10 +8,15 @@
 
 GlobalRegistry = Registry()
 
+logger = logging.getLogger("spectator.init")
+
 try:
     from spectatorconfig import auto_start_global
+
     if auto_start_global():
         GlobalRegistry.start()
         atexit.register(GlobalRegistry.stop)
-except:
-    pass
+    else:
+        logger.warning("module spectatorconfig auto-start is disabled - GlobalRegistry will not start")
```

Comment:
I would make this debug or info at the highest. If auto-start is legitimately disabled, then the user shouldn't get a regular warning.

Reply:
Agree with moving it to debug.